### DISCUSSION TITLE
Intersphinx fails with sqlalchemy

### DISCUSSION THREAD

**Discussion by kasium** [sqlalchemy/sqlalchemy#11609] [Usage Questions] [answered]
When using the latest sphinx version (7.4.0) with an intersphinx mapping to sqlalchemy, the following error/warning occurs:

failed to reach any of the inventories with the following issues:
intersphinx inventory 'http://docs.sqlalchemy.org/en/stable/objects.inv' not readable due to SphinxWarning: inventory <https://docs.sqlalchemy.org/en/20> contains multiple definitions for std:term:isolation

**Comment by zzzeek** [comment, marked as answer]
hi -

Mostly I have nothing to offer here since no such warning is created when I generate with 7.4.3 locally.   I do notice in our glossary we have "isolation" defined with both uppercase and lowercase spellings as synonyms, I've pushed out a change 4cc32ae1f730af19c that will be on the site in an hour that removes the uppercased form of it.

not really sure why Sphinx would refuse to read an inventory with such a small issue though, a warning should not really be a stopper so perhaps change your warnings filters in whatever application you're using

**Reply by kasium** [reply]
Thanks a lot!

**Reply by zzzeek** [reply]
did that fix it ?

**Reply by kasium** [reply]
yes! Thanks a lot
